ui/index.js

```diff
@@ -103,10 +103,13 @@ module.exports = settings => {
   app.use(sendResponse(settings));
   app.use(errorHandler());
 
-  return {
+  const _app = (...args) => app(...args);
+
+  return Object.assign(_app, {
```

Comment:
What's this doing? I'm sure it's something clever but I can't see the use case

Reply:
In order to be able to mount the app onto another app (which we do in order to be able to wrap the app in a user mock) it needs to look like a middleware - i.e. be a function that takes `req, res, next` as arguments.

However, we can't just return the app itself, because we need `app.use` externally to actually mount onto the router.

So instead we create a function that proxies the app itself, and then assign it some custom methods - some of which just pass through to the app.

Reply:
nice! 👯‍♂️